kafka-streams/src/test/groovy/io/micronaut/configuration/kafka/streams/uncaught/MyStreamsUncaughtExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package io.micronaut.configuration.kafka.streams.uncaught;
+
+// tag::imports[]
+import io.micronaut.configuration.kafka.streams.event.BeforeKafkaStreamStart;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.event.ApplicationEventListener;
+import jakarta.inject.Singleton;
+import org.apache.kafka.streams.errors.StreamsUncaughtExceptionHandler;
+// end::imports[]
+
+@Requires(property = "do.not.enable", value = "just for documentation purposes")
```

Comment:
Annotate with `@Requires(property = "spec.name"` instead

Reply:
This one needs to be always disabled. It's there just to make sure that the snippet provided in the documentation compiles.

Reply:
should not be enable for a test?

Reply:
Not really; this snippet is just the ["current workaround"](https://github.com/micronaut-projects/micronaut-kafka/issues/424) and it's here just to illustrate what can be achieved via new config `uncaught-exception-handler`.

Reply:
Please add a test. If we recommend people they can use this, I want a test which verifies it works as described.

Reply:
Done https://github.com/micronaut-projects/micronaut-kafka/pull/780/commits/a7a9e31f13b71c89479df5a31433c3440be2741c